nemo/collections/llm/gpt/model/base.py

```diff
@@ -62,6 +64,18 @@ def gpt_forward_step(model, batch) -> torch.Tensor:
     return model(**forward_args)
 
 
+def transformer_engine_layer_spec(config: "GPTConfig") -> ModuleSpec:
+    return gpt_layer_specs.get_gpt_layer_with_transformer_engine_spec(
+        num_experts=config.num_moe_experts, moe_grouped_gemm=config.moe_grouped_gemm, qk_layernorm=config.qk_layernorm
+    )
+
+
+def local_layer_spect(config: "GPTConfig") -> ModuleSpec:
```

Comment:
should these functions be exposed in `nemo/collections/llm/__init__.py`? just wondering how a user would choose between these two.

Reply:
Yeah at the very least we should expose it inside `llm.gpt`. I can add that tomorrow